src/router.py

```diff
@@ -0,0 +1,159 @@
+from router_leg import RouterLeg    
+import pascy
+import socket
+import select
+from functools import lru_cache
+
+ROUTER_LEGS = []
+
+ARP_TABLE = {'1.1.1.2':'02:42:01:01:01:02',
```

Comment:
What about MAC addresses of the router itself?

Reply:
This is the leg's MAC